### DISCUSSION TITLE
Can we expect Real-time Compositor compatibility (currently in alpha)?

### DISCUSSION THREAD

**Discussion by VelocityRa** [bnpr/Malt#431] [Q&A]
The [Real-time Compositor](https://devtalk.blender.org/t/real-time-compositor-feedback-and-discussion/25018) patchset has finally been merged into Blender master and it obviously meshes really well with NPR workflows, so when can we expect Malt to be compatible with these builds?

Do we have to wait till 3.4 release? I would use this even if it's just a WIP branch.

**Comment by pragma37** [comment]
> Do we have to wait till 3.4 release?

TLDR; Yes.

Malt needs to include a small part of the Blender C source code in its releases and it needs to be in sync with the Blender build used.
This means that even if I ship a 3.4 version today, it doesn't mean it will remain compatible with future 3.4 alpha builds.

In any case, Malt already has its own real-time compositing workflow through Render Nodes, and while it lacks the built-in nodes that Blender has, you can build your own using Screen Shaders.

**Reply by VelocityRa** [reply]
Right, I was hoping it'd just be pinned to a specific commit. But that's completely fair, and yeah malt nodes are more powerful anyway (in most ways).
Thanks for the reply and your work!
